x-pack/plugins/uptime/public/apps/plugin.ts

```diff
@@ -87,6 +87,28 @@ export class UptimePlugin
       order: 8400,
       title: PLUGIN.TITLE,
       category: DEFAULT_APP_CATEGORIES.observability,
+      meta: {
+        keywords: [
+          'Synthetics',
+          'pings',
+          'checks',
+          'availability',
+          'response duration',
+          'response time',
+          'outside in',
+          'reachability',
+          'reachable',
+          'digital',
```

Comment:
Same comment regarding `digital` as the one I had in the UX keywords.

Reply:
Same as above:

Generality was certainly what I was going for with this. With digital tied to so many things: digital performance, digital experience, digital transformation, I wanted to try and be general at the top of the funnel, so to speak, for discoverability reasons. cc @justinkambic